.readme-partials.yml

```diff
@@ -25,3 +25,29 @@ body: |-
 
   The `@google-cloud/logging` library will handle batching and dispatching
   these log lines to the API.
+
+  ## Writing to Stdout
+
+  Serverless applications like Cloud Functions, Cloud Run, and App Engine
+  should use the `LogSync` class to synchronously log to `stdout` or another
+  custom transport. Logs written to `stdout` are picked up by a [Logging agent](https://cloud.google.com/logging/docs/agent/logging)
+  injected with additional context, and streamed to the Logging API.
+
+  This library's `LogSync` gives users idiomatic syntax sugar, an on/off toggle
+  between writing to the API vs to `stdout`, and extracts additional log
+  properties like trace context from HTTP headers.
```

Comment:
does it provide the methods for logging at different verbosity levels? e.g. Log.info, Log.warning, etc?

Reply:
yes it does! lemme add that